pkg/deviceshifu/deviceshifuhttp/deviceshifuhttp.go

```diff
@@ -232,12 +232,12 @@ func (handler DeviceCommandHandlerHTTP) commandHandleFunc() http.HandlerFunc {
 			}
 
 			rawRespBodyString := string(respBody)
-			_, shouldUsePythonCustomProcessing := deviceshifubase.CustomInstructionsPython[handlerInstruction]
+			InstructionFuncName, shouldUsePythonCustomProcessing := deviceshifubase.CustomInstructionsPython[handlerInstruction]
```

Comment:
let's not use capitalized `I` for variable names that are not being exported, same for other occurrences.